### PR TITLE
Require disp-table before attempting to access standard-display-table

### LINKED ISSUE #28: wrong-type-argument char-table-p
If I byte compile Nano, I get the following error when loading `nano-layout`:

```
Debugger entered--Lisp error: (wrong-type-argument char-table-p nil)
  set-char-table-extra-slot(nil 0 (8230 . 101))
  set-display-table-slot(nil truncation (8230 . 101))
  byte-code("\306\307\310\311\312\313\314\315\316\317\320\321\257\13!\20\322\323\324\325#\210\326\11\327\330\331\323\"#\210\326\11\332\330\333\323\"#\210\334\335\336\"\210\337\211\22\23\337..." [default-frame-alist standard-display-table inhibit-startup-screen inhibit-startup-message inhibit-startup-echo-area-message initial-scratch-message append (font . "Roboto Mono:style=Light:size=14") (min-height . 1) (height . 45) (min-width . 1) (width . 81) (vertical-scroll-bars) (internal-border-width . 24) (left-fringe . 0) (right-fringe . 0) (tool-bar-lines . 0) (menu-bar-lines . 0) custom-declare-face fallback ((t :family "Fira Code" :inherit 'nano-face-faded)) "Fallback" set-display-table-slot truncation make-glyph-code 8230 wrap 8617 add-hook term-mode-hook #f(compiled-function () #<bytecode 0x1fe646e81999>) t nil tool-bar-mode 0 tooltip-mode menu-bar-mode global-hl-line-mode 1 24 right-only window-divider-mode provide nano-layout x-underline-at-descent-line window-divider-default-right-width window-divider-default-places widget-image-enable org-hide-emphasis-markers] 13)
  require(nano-layout)
  eval-buffer(#<buffer  *load*> nil "/Users/ndw/testemacs/.emacs.d/init.el" nil t)  ; Reading at buffer position 92
  load-with-code-conversion("/Users/ndw/testemacs/.emacs.d/init.el" "/Users/ndw/testemacs/.emacs.d/init.el" t t)
  load("/Users/ndw/testemacs/.emacs.d/init" noerror nomessage)
  startup--load-user-init-file(#f(compiled-function () #<bytecode 0x1fe646e7f5a9>) #f(compiled-function () #<bytecode 0x1fe646e7f5c1>) t)
  command-line()
  normal-top-level()
```

I'm running this in a test home directory (`HOME=/Users/ndw/testemacs emacs`) with this init file:

```
(add-to-list 'load-path "/Users/ndw/testemacs/.emacs.d/nano-emacs")

(require 'nano-layout)
```

The `nano-emacs` directory is just what I cloned from GitHub, with the files byte compiled. I have no trouble loading `nano-layout` if I delete the `.elc` file.

### PR DESCRIPTION
**Pull request by ndw**
This PR fixes #28. It was suggested in email following my Emacs bug report.

### CODE REVIEW

File: nano-layout.el [hunk 1]
```diff
@@ -16,9 +16,11 @@
 ;; along with this program. If not, see <http://www.gnu.org/licenses/>.
 ;; ---------------------------------------------------------------------
 
+(require 'disp-table)
+
 (setq default-frame-alist
       (append (list
-	       '(font . "Roboto Mono:style=Light:size=14")
+	       '(font . "Roboto Mono:style=Medium:size=18")
```
Comment (rougier) [inline nano-layout.el:23]: Can you put back default size/weight?

Comment (ndw) [reply nano-layout.el]: Yes. Done. Sorry about that. Careless of me.

**Comment by rougier** [comment]
Sorry for the delay. Could you rebase your PR?

**Comment by ndw** [comment]
Done

**Comment by rougier** [comment]
Thanks. The diff on GitHub still indicates large change Did you push your rebase?

**Comment by ndw** [comment]
Near as I can tell, yes:
```
$ git diff upstream/master
diff --git a/nano-layout.el b/nano-layout.el
index 47fe9a3..1e53ef4 100644
--- a/nano-layout.el
+++ b/nano-layout.el
@@ -16,6 +16,8 @@
 ;; along with this program. If not, see <http://www.gnu.org/licenses/>.
 ;; ---------------------------------------------------------------------

+(require 'disp-table)
+
 (setq default-frame-alist
       (append (list
               '(font . "Roboto Mono:style=Light:size=14")

nano-emacs on  fix28 [$]
$ git push
Everything up-to-date
```

**Review by brongulus — COMMENTED** [review]
I don't think the rebase was successful, it still shows a large diff. By the way, I did check the change on my end and adding `(require 'disp-table)` does solve the initial issue. Thanks for figuring it out @ndw

**Comment by rougier** [comment]
@brongulus Separate issue for the menu-bar would be nice yes.
@ndw I4ve no idea why GitHub reports such large change to be made. Could you try a new PR just to be sure?

**Comment by ndw** [comment]
Right. I don't know either. I'll make new PRs today.

**Comment by ndw** [comment]
Superceded by #72.

**Pull request closed without merging**